Bottom: dad8b0b119a7be6f828ab02f8a74072c0656f5e4
Top:    55d96083299209962f94cc22bc131297440fcbcb
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-04-26 01:35:08 +0100

Refresh of demo-of-exporter-specific-expo

---

diff --git a/src/draw.c b/src/draw.c
index d2c4cd4..1495526 100644
--- a/src/draw.c
+++ b/src/draw.c
@@ -83,8 +83,6 @@ static const BoxType *clip_box = NULL;
  * some local prototypes
  */
 static void DrawEverything (BoxTypePtr);
-static void DrawPPV (int group, const BoxType *);
-static int DrawLayerGroup (int, const BoxType *);
 static void AddPart (void *);
 static void SetPVColor (PinTypePtr, int);
 static void DrawEMark (ElementTypePtr, LocationType, LocationType, bool);
